integration_tests/testsuite/test_util.py

```diff
@@ -97,7 +115,7 @@ def _post(url, payload, timeout=DEFAULT_TIMEOUT):
     except requests.exceptions.Timeout:
         logging.error('POST to {0} timed out after {1} seconds!'
                       .format(url, timeout))
-        return 1
+        return 'ERROR', 1
 
 
 def _check_response(response, error_message):
```

Comment:
the reason i did this is so we can always check the response from the sample application when we make a POST request, specifically so that it can return the log location back to the test driver. I'm pretty confident with the way I've implemented it, let me know if you see issues with it though